develop/tutorials/articles/100-tooling/04-liferay-workspace/08-leveraging-docker-in-liferay-workspace.markdown

```diff
@@ -50,9 +50,11 @@ To find the possible property values you can set, see the official @product@
 Docker Hub's Tags section (e.g.,
 [Liferay Portal Docker Tags](https://hub.docker.com/r/liferay/portal/tags).
 
+The container will be given the name `$projectName-liferayapp`.
+
 Lastly, a new `build/docker` folder is generated in your workspace. This folder
-bridges your local file system to the container's file system. This means files
-in workspace's `build/docker` folder are also available in the container's
+will be mounted from your local file system to the container's file system. This
```

Comment:
@david-truong according to our writing guidelines, we try to avoid using *will be* unless it is absolutely necessary. The *mount* wording is a bit foreign to me. Do you think we could say this instead:?
> This folder is mounted into the container's file system.

Let me know if there's a better active voice way to describe that.

Reply:
That's okay too